src/Jackett.Common/Definitions/nicept.yml

```diff
@@ -175,5 +174,8 @@ search:
       text: 259200
     description:
       selector: td.rowfollow:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
+      filters:
+        - name: replace
+          args: ["剩余时间：", ""]
 # NexusPHP v1.7.29 2022-10-13
```

Comment:
check if profile allows english

Reply:
hddolby, nicept, and ourbits all have a language profile setting, but it's per session rather than permanent, so Jackett will always get the default.

Reply:
ok, but if you are having to strip the chinese 'will end in' from the description then you have removed the freeleech will expire timestamp too, which was a useful piece of the description that was available before.